plants-backend-adapter-service/src/test/resources/commodity/UsedFarmMachinery.json

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
Is this being used anywhere? I thought we are using one combined schema for all commodities?